.github/CODEOWNERS

```
@@ -231,18 +243,18 @@
 /releasenotes-installscript/            @DataDog/documentation
 /releasenotes-dca/                      @DataDog/documentation @DataDog/container-integrations @Datadog/container-app
 
-/rtloader/                              @DataDog/agent-core
+/rtloader/                              @DataDog/agent-shared-components
```

Since it's tied to the collector and python checks

```suggestion
/rtloader/                              @DataDog/agent-metrics-logs
```